packages/stow.rb

```diff
@@ -9,6 +9,12 @@ class Stow < Package
   source_url 'https://ftpmirror.gnu.org/stow/stow-2.3.1.tar.gz'
   source_sha256 '09d5d99671b78537fd9b2c0b39a5e9761a7a0e979f6fdb7eabfa58ee45f03d4b'
 
+  depends_on 'perl'
```

Comment:
This should really just be named perl_stow. Maybe I should just make that change, since this is a perl module?

I added the dependency so that people would notice when rebuilding all perl packages.